src/pivotal-ui/components/header/header.scss

```diff
@@ -8,8 +8,34 @@ categories:
  - css_base_header
  - css_all
 ---
+
+To use the helper providing the header:
+
+```
+{{fromPackage "@npmcorp/pui-css-header/header.hbs" context}}
+```
+
+Context requirements:
+
+- `user` object: If present, includes user avatar and internal navigation menu, otherwise presents login links and sign-in links.
+  - `name` property: the user's name
+  - `isPaid` property: boolean indicating whether the user is paid
+  - `avatar` object: URLs for each size of avatar (this partial uses `small`)
+- `env` object: Configuration from the environment of relevance to the display layer
+  - `NPMO_COBRAND` propery: A string to display in Enterprise mode
+  - `CANONICAL_HOST` property: URL root of this site to form self-referential URLS
+- `features` object: Feature flags
+  - 'npmo' property: If true, uses the Enterprise header.
+- `q` property: The current search terms
+
+Requirements:
+
+- `handlebars-helper-svg` helper
+- `form-security` partial
+
```

Comment:
YES I DO!

Wow, I searched for `{{` and managed to gloss over those repeatedly.